tools/deploy.sh

```diff
@@ -229,6 +229,10 @@ if [[ "${DEPLOY_BMO}" == "true" ]]; then
     pushd "${TEMP_BMO_OVERLAY}"
     ${KUSTOMIZE} create --resources=../../base,../../namespace \
     --namespace=baremetal-operator-system
+    # Disable the name suffix hash for generated resources.
+		# facilitating better compatibility with development tools like Tilt
+    echo "generatorOptions:" >> kustomization.yaml
```

Comment:
Please do multiline addition instead of echoing twice.

```suggestion
    cat <<EOF >>kustomization.yaml
generatorOptions:
  disableNameSuffixHash: true
EOF
```